src/main/java/org/traccar/config/Keys.java

```diff
@@ -542,6 +542,13 @@ private Keys() {
             "database.saveEmpty",
             List.of(KeyType.CONFIG));
 
+    /**
+     * This property controls the maximum lifetime in miliseconds of a connection in the pool.
+     */
+    public static final ConfigKey<Integer> DATABASE_MAX_LIFETIME = new IntegerConfigKey(
+            "database.maxLifetime",
+            List.of(KeyType.CONFIG),60000);
```

Comment:
What is the default value? Can you please link to the documentation.

Reply:
@tananaev  yep, your right, it's a little bit different value according to [docs](https://github.com/brettwooldridge/HikariCP#gear-configuration-knobs-baby). Fixed.